types/isaac/isaac-tests.ts

```diff
@@ -0,0 +1,8 @@
+import isaac = require('isaac');
```

Comment:
the exports are ES syntax, so the import should use ES syntax: `import * as isaac from 'isaac'`

Reply:
fixed now